packages/common/src/hardforks/shanghai.json

```diff
@@ -3,5 +3,5 @@
   "comment": "Next feature hardfork after the merge hardfork having withdrawals, warm coinbase, push0, limit/meter initcode",
   "url": "https://github.com/ethereum/pm/issues/356",
   "status": "Pre-Draft",
-  "eips": [4895, 3651, 3855, 3860]
+  "eips": [4895]
```

Comment:
TODO: remember to undo this while merging

Reply:
Should we really undo this?  Seems like maybe we should wait till we have the final inclusion list for Shanghai (beyond 4895) before merging it back in, maybe?

Reply:
we should, for eip4844 we can keep a branch open with these/some of the other openended things